CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ You can find its changes [documented below](#083---2023-02-28).
 
 ### Added
 
+- Type name is now included in panic error messages in WidgetPod ([#2380] by [@matthewgapp])
```

Comment:
Just some minor adjustments for the changelog.

```suggestion
- Type name is now included in panic error messages in `WidgetPod`. ([#2380] by [@matthewgapp])
```

Also please define both `#2380` and `@matthewgapp` at the bottom of the changelog. Unfortunately these aren't automatic.

Reply:
I defined the PR number and my username in the changelog.